pkg/url/kubernetes_test.go

```diff
@@ -33,7 +34,7 @@ import (
 func getFakeURL(name string, host string, port int, path string, protocol string, kind localConfigProvider.URLKind, urlState StateType) URL {
 	return URL{
 		TypeMeta: v1.TypeMeta{
-			Kind:       "url",
+			Kind:       "URL",
```

Comment:
How about using `URLKind` defined in url/types.go? Or is this intended?

Reply:
In tests, I like to have the constant value, so the test also validates the `URLKind` value